### PR TITLE
Database: Added missing warrior action bar buttons.

### PR DESCRIPTION
**Pull request by zTerragor**
Database: Added missing warrior action bar buttons. взято с skyfire

### CODE REVIEW

**Comment by leelf00** [comment]
Have you tested in game?

**Comment by zTerragor** [comment]
> Have you tested in game?

yes works on sky fire

**Comment by leelf00** [comment]
We do not have playercreateinfo_spell_cast

File: sql/updates/master/world/2023_11_07_world_00.sql [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+(7, 4, 9, 20589, 0),
+(7, 5, 9, 20589, 0),
+(7, 6, 9, 20589, 0),
+(7, 8, 9, 20589, 0),
+(7, 9, 9, 20589, 0),
+(7, 10, 9, 20589, 0),
+(7, 10, 81, 20589, 0);
+
+-- Warrior Auto Cast Battle Stance 
+DELETE FROM `playercreateinfo_spell_cast` WHERE `racemask`=0 AND `classmask`=1 AND `Spell`=2457;
```
Comment (leelf00) [inline sql/updates/master/world/2023_11_07_world_00.sql:15]: Core don't have this table

**Comment by leelf00** [comment]
Fixed by https://github.com/Legends-of-Azeroth/Legends-of-Azeroth-Pandaria-5.4.8/pull/268

**Pull request closed without merging**